conmon/conmon.c

```diff
@@ -354,19 +351,19 @@ static int write_k8s_log(int fd, stdpipe_t pipe, const char *buf, ssize_t buflen
 		}
 
 		/* Output the timestamp */
-		if (writev_buffer_append_segment(fd, &bufv, tsbuf, -1) < 0) {
+		if (writev_buffer_append_segment(fd, &bufv, tsbuf, 43) < 0) {
```

Comment:
I'd prefer to see 43 defined to some constant, but your call.

Reply:
Fixed.